coalib/parsing/FilterHelper.py

```diff
@@ -59,9 +60,9 @@ def apply_filters(filters, bears=None, sections=None):
     and runs filter in bear filtering mode.
 
     :param filters:
-        List of args based on ``bears`` has to be filtered. For example:
-        ``[['language', 'c', 'java'], ['can_fix', 'syntax'],
-        ['section_tags', 'save']]``
+        Dict of args based on ``bears`` has to be filtered. For example:
+        ``{'language': ('c', 'java'), 'can_fix': ('syntax',),
```

Comment:
Since we are using `OrderedDict`'s you should update the docs to show that. Please go over all the docs and update this.